src/test/scala/uk/gov/nationalarchives/tdr/api/routes/ConsignmentRouteSpec.scala

```diff
@@ -153,15 +157,58 @@ class ConsignmentRouteSpec extends AnyFlatSpec with Matchers with TestRequest wi
     response.errors.head.message should equal(expectedResponse.errors.head.message)
   }
 
-  private def checkConsignmentExists(consignmentId: UUID): Unit = {
+  "updateExportLocation" should "update the export location correctly" in {
+    insertConsignment()
+    val prefix = "json/updateexportlocation_"
+    val expectedResponse = getDataFromFile[GraphqlMutationExportLocation](prefix)("data_all")
+    val token = validBackendChecksToken("export")
+    val response: GraphqlMutationExportLocation = runTestRequest[GraphqlMutationExportLocation](prefix)("mutation_all", token)
+    response.data should equal(expectedResponse.data)
+    getConsignmentField(UUID.fromString("6e3b76c4-1745-4467-8ac5-b4dd736e1b3e"), "ExportLocation") should equal("6e3b76c4-1745-4467-8ac5-b4dd736e1b3e.tar.gz")
+  }
+
+  "updateTransferInitiated" should "update the transfer initiated date correctly" in {
+    insertConsignment()
+    val prefix = "json/updatetransferinitiated_"
+    val expectedResponse = getDataFromFile[GraphqlMutationTransferInitiated](prefix)("data_all")
+    val response: GraphqlMutationTransferInitiated = runTestRequest[GraphqlMutationTransferInitiated](prefix)("mutation_all", validUserToken())
+    response.data should equal(expectedResponse.data)
+    val field = getConsignmentField(UUID.fromString("6e3b76c4-1745-4467-8ac5-b4dd736e1b3e"), _)
+    Option(field("TransferInitiatedDatetime")).isDefined should equal(true)
+    field("TransferInitiatedBy") should equal("4ab14990-ed63-4615-8336-56fbb9960300")
+  }
+
+  private def insertConsignment(): Unit = {
```

Comment:
Does `TestUtils.createConsignment` do the same thing?

Reply:
It does indeed, cheers. I've changed it to use that.